build/server-image/kube-apiserver/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG arch
```

Comment:
@BenTheElder - I added a Dockerfile like you suggested. Hopefully this looks better than before.

Reply:
BUILDX actually sets TARGETARCH for this.
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

Reply:
Updated to use `TARGETPLATFORM`.